src/host/buffered_io.c

```diff
@@ -9,7 +9,7 @@
 #include "premake.h"
 #include "buffered_io.h"
 
-void buffer_init(Buffer* b)
+void buffer_initialize(Buffer* b)
```

Comment:
Not sure why this was renamed?

Reply:
luasocket has a function with the same name, and C doesn't allow function aliases as I figured out (I'm used to C++ so that took me by surprise) so I renamed as little as possible